core/src/main/scala/cats/data/NonEmptySet.scala

```diff
@@ -417,3 +420,29 @@ sealed abstract private[data] class NonEmptySetInstances {
     def combine(x: NonEmptySet[A], y: NonEmptySet[A]): NonEmptySet[A] = x | y
   }
 }
+
+sealed abstract private[data] class NonEmptySetInstances0 extends NonEmptySetInstances1 {
+  implicit def catsDataOrderForNonEmptySetReal[A](implicit A: Order[A]): Order[NonEmptySet[A]] =
```

Comment:
nit, is that `Real` suffix still needed?

Reply:
Not sure. catsDataOrderForNonEmptySet is the name of the existing one (which I removed the `implicit` keyword from). I can play around with it and see if it breaks bin compat

Reply:
did you mean `private[data] def catsDataEqForNonEmptySet[A: Order]:` ?

Reply:
yep since `NonEmptySetInstances` extends `NonEmptySetInstances0`

Reply:
:) `catsDataOrderForNonEmptySet` is different from `catsDataEqForNonEmptySet`.

Reply:
🤦‍♂️ well if MiMa is happy then I'm happy :)
I'm gonna try remove the old `catsDataEqForNonEmptySet` and see what happens

Reply:
sorry I wasn't clear. I meant that you didn't need to add a `Real` suffix, because it didn't conflict with anything else. It's the right approach to remove the `implict` keyword for `catsDataEqForNonEmptySet` , but you didn't need to add private[data] since the class is already `private[data]`

Reply:
yes I got you. What I was also going to do is to remove the original `catsDataEqForNonEmptySet`, since there's already a method named the same in `NonEmptySetInstances1`. (I think JVM's use of invokevirtual call means it won't break bin compat - so it'll be both source and binary compatible)

Reply:
Done